tests/provider/dwd/observation/test_metaindex.py

```diff
@@ -26,7 +27,17 @@ def test_meta_index_creation():
 
     assert not meta_index.empty
 
-    with pytest.raises(requests.exceptions.HTTPError):
+
+@pytest.mark.remote
+def test_meta_index_creation_failure():
+
+    with pytest.raises(
+        (
+            requests.exceptions.HTTPError,
+            aiohttp.client_exceptions.ClientResponseError,
```

Comment:
Same as for fileindex I think we should only allow for FIleNotFoundError

Reply:
Fixed with 6d658c983.